dot/core/service_integration_test.go

```diff
@@ -873,78 +997,3 @@ func TestDecodeSessionKeys_WhenGetRuntimeReturnError(t *testing.T) {
 	require.Error(t, err, "problems")
 	require.Nil(t, b)
 }
-
-func TestGetReadProofAt(t *testing.T) {
-	keysToProof := [][]byte{[]byte("first_key"), []byte("another_key")}
-	mockedProofs := [][]byte{[]byte("proof01"), []byte("proof02")}
-
```

Comment:
Why are these tests gone? Are they moved somewhere else?

Reply:
These were using mocks and struck me as unit tests. Yup the respective tests exists as part of the unit tests

Reply:
Ah ok this makes sense, thanks!